src/spatio_temporal_voxel_layer.cpp

```diff
@@ -249,6 +252,11 @@ void SpatioTemporalVoxelLayer::onInitialize(void)
           boost::bind(&SpatioTemporalVoxelLayer::PointCloud2Callback, this, _1, \
                                                    _observation_buffers.back()));
 
+      boost::shared_ptr <ros::ServiceServer> server(new ros::ServiceServer);
```

Comment:
Please test this and make sure it does what you expect it to. Can you add an example in the readme file of how to use this to turn on/off the observation sources (and what topics/service names they come up under)?

Generally speaking, it looks clean, good work